test/e2e/specs/dashboard/settings__site-visibility.ts

```diff
@@ -0,0 +1,86 @@
+/**
+ * @group calypso-pr
+ * @group dashboard
+ * @group settings
+ */
+
+import {
+	TestAccount,
+	getTestAccountByFeature,
+	envToFeatureKey,
+	envVariables,
+	DashboardPage,
+} from '@automattic/calypso-e2e';
+import { Page, Browser } from 'playwright';
+
+declare const browser: Browser;
+
+/**
+ * Tests that a user can edit privacy settings on a site via the dashboard.
+ *
+ * This test verifies that site visibility (privacy) options can be changed
+ * between Public, Private and Coming Soon modes.
+ */
+describe( 'Dashboard: Site Visibility Settings', function () {
```

Comment:
Is it intentional that we don't have a test for the `Public` visibility? I.e. verifying that we can indeed view the site.

Maybe we can add assertion that it does NOT contain `coming soon` nor `private` in the frontend.

Reply:
I can add one :)